### ISSUE TITLE
Fix parentheses in list comprehensions

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8351]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8351|
|Type|enhancement|
|Created|2016-05-24 09:27:09Z|
|Branch|https://github.com/twisted/twisted/tree/8351-rodrigc-paren|

From this example: http://bugs.python.org/issue2367

In Python 2, this code is legal:
```
 [x for x in 1, 2]
```

but it is a syntax error in Python 3.
The code needs to be fixed in this way:

```
 [x for x in (1, 2)]
```

Some of these cases can be found with:  **2to3 -f paren**

<details><summary>Searchable metadata</summary>

```
trac-id__8351 8351
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8351_rodrigc_paren 8351-rodrigc-paren
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1464082029542710 1464082029542710
changetime__1464126422580497 1464126422580497
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/70

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

The change looks good.

I added the topfile.

I have no idea what other strategy to use to detect all this invalid cases.. so I guess that the code was pretty clean of this issue.

Hope the builders are green and I can merge it.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset b7628ff79dbcd88d19ad9e944659fc5f84b54e3b
```
#!CommitTicketReference repository="" revision="b7628ff79dbcd88d19ad9e944659fc5f84b54e3b"
Merge 8351-rodrigc-paren: Update parentheses in list comprehensions for py3 syntax.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8351
```
